aim/web/ui/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -19,20 +20,23 @@ function ProgressBar({
   React.useEffect(() => {
     if (processing || pendingStatus) {
       setRenderBar(true);
+      setIsProgressBarVisible?.(true);
     } else {
       const hidingDelay = 2000;
       if (timeoutIdRef.current) {
         window.clearTimeout(timeoutIdRef.current);
       }
       timeoutIdRef.current = window.setTimeout(() => {
         setRenderBar(false);
+        setIsProgressBarVisible?.(false);
       }, hidingDelay);
     }
     return () => {
       if (timeoutIdRef.current) {
         window.clearTimeout(timeoutIdRef.current);
       }
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
remove es-lint disable row please and include `setIsProgressBarVisible` in dependency array